benches/torii.rs

```diff
@@ -1,40 +1,29 @@
 use criterion::*;
-use iroha::{
-    client::query::Query,
-    config::Configuration,
-    consensus::sumeragi::Sumeragi,
-    model::{block::Blockchain, tx::Transaction},
-    networking::torii::Torii,
-    storage::kura::Kura,
-};
+use futures::executor;
+use iroha::{client::query::GetAccountAssets, config::Configuration, prelude::*};
 use std::io::prelude::*;
+use std::thread;
 use std::time::Duration;
 
 const QUERY_REQUEST_HEADER: &[u8; 16] = b"GET / HTTP/1.1\r\n";
 const COMMAND_REQUEST_HEADER: &[u8; 25] = b"POST /commands HTTP/1.1\r\n";
 const _PUT: &[u8; 16] = b"PUT / HTTP/1.1\r\n";
 const OK: &[u8; 19] = b"HTTP/1.1 200 OK\r\n\r\n";
+static DEFAULT_BLOCK_STORE_LOCATION: &str = "./blocks/";
 
 fn query_requests(criterion: &mut Criterion) {
-    let config = Configuration::from_path("config.json").expect("Failed to load configuration.");
-    let torii_url = config.torii_url.to_string();
-    std::thread::spawn(move || {
-        let mut torii = Torii::new(
-            &torii_url,
-            Sumeragi::new(Blockchain::new(
-                futures::executor::block_on(Kura::strict_init()).expect("Failed to init Kura."),
-            )),
-        );
-        torii.start();
-    });
+    thread::spawn(|| executor::block_on(create_and_start_iroha()));
```

Comment:
do we need block on create_and_start_iroha() or can we just assume that iroha is started?

Reply:
We `block_on` in a different thread.
We use this method to give `create_and_start_iroha` method an async context.
I didn't place `block_on` inside  `create_and_start_iroha` to prevent double context creation and panic in this case.